docs/source/user_guide/computed_properties.rst

```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
 
 .. _mobility:
 
-.. Charge Carrier Mobility
-.. -----------------------
+Charge Carrier Mobility
+-----------------------
+
+Charge carrier mobility is a measure of the speed at which charge carriers, such as electrons or holes,
+can move through a material when an electric field is applied. It reflects the material's ability to conduct charge and
+is influenced by various intrinsic and extrinsic factors.
+
+
+Modeling Method
+~~~~~~~~~~~~~~~
+`Nanoscope` estimates the electron and hole mobility using the Generalized Effective Medium Model (GEMM) method
+using the formula:
+
+.. math::
+
+   \mu = \frac{e \cdot \beta \cdot M \cdot J^2 r^2}{n \cdot \hbar \cdot \sqrt{\lambda}}
+         \cdot \sqrt{\frac{\pi \cdot \beta}{1 + \frac{\beta \cdot \sigma^2}{\lambda}}}
+         \cdot \exp\left(-C \cdot (\beta \cdot \sigma)^2 - C \cdot \beta \cdot \lambda\right)
+
+where constants are:
+
+- :math:`e` is the elementary charge,
+- :math:`\beta = \frac{1}{k_B \cdot T}` with :math:`k_B` being the Boltzmann constant and :math:`T` being the inverse temperature,
+- :math:`n` is the dimensionality of the system,
+- :math:`\hbar` is the reduced Planck's constant,
+
+Other parameters are explained below.
+
+Fixed Parameters
```

Comment:
Subsection of "Modeling Method"? If so, header should be one level lower, same for Computed Parameters

Reply:
yes. fixed